package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "test": "npm run lint && npm run unit-tests",
     "lint": "eslint './src/**/*.js'",
-    "unit-tests": "jest --ci -w=2"
+    "unit-tests": "jest --ci -w=2",
+    "generate-docs": "./node_modules/.bin/jsdoc2md ./src/index.js > ./docs/SDK.md"
```

Comment:
You don't need to add the path `./node_modules/.bin/` here, npm will find the bin

Reply:
done